pkgs/by-name/op/opencollada-blender/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  libxml2,
+  pcre,
+  darwin,
+}:
+
+stdenv.mkDerivation {
+  pname = "opencollada";
+
+  version = "unstable-2025-01-30";
+
+  src = fetchFromGitHub {
+    owner = "aras-p";
+    repo = "OpenCOLLADA";
+    rev = "4526eb8aaa6462c71fbedd23103976c151a01c50";
+    sha256 = "sha256-ctr+GjDzxOJxBfaMwjwayPkAOcF+FMsP1X72QCOwvTY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin (
+    with darwin.apple_sdk.frameworks; [ AGL ]
+  );
+
```

Comment:
```suggestion
```
Our darwin SDK has been reworked, so these frameworks are not needed and provide empty dummies.